picard/formats/id3.py

```diff
@@ -292,16 +292,25 @@ def _load(self, filename):
                         metadata.add(name, text)
             elif frameid == "TMCL":
                 for role, name in frame.people:
+                    if role == 'performer':
+                        role = ''
                     if role or name:
                         metadata.add('performer:%s' % role, name)
+                    elif name:
+                        metadata.add('performer', name)
```

Comment:
Something looks wrong here, won't the `elif` never run since the above `if` already covers the case that there is a name? I think this should be:
```suggestion
                    if role and name:
                        metadata.add('performer:%s' % role, name)
                    elif name:
                        metadata.add('performer', name)
```

Reply:
Indeed, thanks for spotting. Actually this can be simplified, no need to check if name is non empty. `metadata.add` takes care of that properly, so the code can be the same as in the "TIPL" case below. Fixed.

Reply:
@kellnerd good catch, thanks